XcodeServerSDKTests/TestUtils.swift

```diff
@@ -56,4 +58,87 @@ extension XCTestCase {
         let configuration = bot.configuration
         return configuration
     }
+}
+
+// MARK: Exception assertions
+// Based on: https://forums.developer.apple.com/thread/5824
```

Comment:
Everyhing thanks to [@modocache](https://twitter.com/modocache) and [@joar_at_work](https://twitter.com/joar_at_work) 😉